### PR TITLE
New version: Lauriethefish.QuestPatcher version 2.7.3

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135092)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Program Files\QuestPatcher\QuestPatcher.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Program Files\QuestPatcher\QuestPatcher.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Program Files\QuestPatcher\createdump.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Program Files\QuestPatcher\createdump.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform. Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\QuestPatcher\QuestPatcher.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\QuestPatcher\QuestPatcher.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform. Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\QuestPatcher\createdump.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\QuestPatcher\createdump.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.

(Automated response - build 612.)

File: manifests/l/Lauriethefish/QuestPatcher/2.7.3/Lauriethefish.QuestPatcher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+UpgradeBehavior: install
+ReleaseDate: 2024-01-16
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/Lauriethefish/QuestPatcher/releases/download/2.7.3/QuestPatcher-windows.exe
+  InstallerSha256: 6506EFE598FACD2CAC769A04BA9DC3D6EFC29B00E7D439655AF4C83B4DECD5DC
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/l/Lauriethefish/QuestPatcher/2.7.3/Lauriethefish.QuestPatcher.installer.yaml:21]:
```suggestion
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/Lauriethefish/QuestPatcher/releases/download/2.7.3/QuestPatcher-windows.exe
  InstallerSha256: 6506EFE598FACD2CAC769A04BA9DC3D6EFC29B00E7D439655AF4C83B4DECD5DC
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
```

Comment (stephengillie) [reply manifests/l/Lauriethefish/QuestPatcher/2.7.3/Lauriethefish.QuestPatcher.installer.yaml:21]: From the errors, I'm guessing this is a 64-bit application.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**